### PR TITLE
feat-added homepage

### PR DESCRIPTION
**Pull request by harshparab**
Created homepage for the e commerce website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/navbar.css">
```
Comment (Snigdha17) [inline index.html:7]: Will you be later using your component library to build the e-commerce site? (as mentioned in lms)

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
+.homepage-content {
+    width: 80%;
+    margin: 3rem auto;    
+}
+
+.homepage-image {
+    width: 100%;
+    border-radius: 0.5rem;
```
Comment (Sharmaryan) [inline css/homepage.css]: For something less than 1 rem use pixels ( upto 10px)

**Review by Snigdha17 — COMMENTED** [review]
Code looks clean - left a comment regarding whether you will be reusing your component library to build the nuclei-store website

File: css/homepage.css (hunk 2 above)
Comment (harshparab) [reply css/homepage.css]: Okay

File: index.html (hunk 1 above)
Comment (harshparab) [reply index.html:7]: Yes I am using it

File: css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
+* {
```
Comment (Sharmaryan) [inline css/navbar.css:1]: Common css can be moved to separate file

**Review by Sharmaryan — APPROVED** [review]

File: css/navbar.css (hunk 3 above)
Comment (harshparab) [inline css/navbar.css]: Okay

Comment (harshparab) [reply css/navbar.css]: Thanks

File: index.html (hunk 1 above)
Comment (Snigdha17) [reply index.html:7]: Okay, You can reference it either in your main stylesheet or in index.html and reuse the components :)

Comment (harshparab) [reply index.html:7]: I forgot completely about it🤦‍♂️, Thanks!

Comment (Snigdha17) [reply index.html:7]: no problem!

Comment (harshparab) [reply index.html:7]: I have not created nav bar in component library, I'll add and later use it

**Review by ishapatell1 — APPROVED** [review]
Your code looks clean and with proper naming conventions. Good work harsh!

**Pull request merged**